src/setOfPeople.js

```diff
@@ -37,8 +37,20 @@
  *
  * @return {Set}
  */
-function setOfPeople(people) {
-  // write code here
+function setOfPeople(people = []) {
+  const uniquePeople = [];
+  const whoStayed = person => {
+    let newPerson = true;
+    if (uniquePeople.includes(person)) {
+      uniquePeople.splice(uniquePeople.indexOf(person), 1);
+      newPerson = !newPerson;
+    } else if (uniquePeople.indexOf(person) === -1 && newPerson) {
+      uniquePeople.push(person);
+    }
+  };
+  people.map(whoStayed);
```

Comment:
What map should and forEach 
https://developer.mozilla.org/uk/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach
https://developer.mozilla.org/uk/docs/Web/JavaScript/Reference/Global_Objects/Array/map

Write me if you still have questions about it